chainer/functions/theano/theano_function.py

```diff
@@ -0,0 +1,149 @@
+import collections
+
+import numpy
+import six
+
+try:
+    import theano
+    import theano.sandbox.cuda as theano_cuda
+    _available = True
```

Comment:
This variable should be exposed to user because I guess user sometimes use this variable, as you do in the unit test.